nibabel/fileslice.py

```diff
@@ -561,7 +564,8 @@ def optimize_read_slicers(sliceobj, in_shape, itemsize, heuristic):
 
 
 def slicers2segments(read_slicers, in_shape, offset, itemsize):
-    """ Get segments from `read_slicers` given input `in_shape` and memory steps
+    """ Get segments from `read_slicers` given input `in_shape` and memory
```

Comment:
Should be one line to match match numpy docstring standard.

Maybe  `Get segments from `read_slicers` given `in_shape` and memory steps` ?